bin/aurelia-cli.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 'use strict';
-const resolve = require('../lib/resolve');
+const resolve = require('resolve');
```

Comment:
@huochunpeng Before I merge this...are we sure this is the only place where `resolve` was used?

Reply:
Yes, I checked.